src/main/java/org/folio/rest/repository/FeeFineActionRepository.java

```diff
@@ -134,24 +134,41 @@ public FeeFineActionRepository(Map<String, String> headers, Context context) {
   public Future<List<Feefineaction>> findActionsByTypeForPeriodAndOwners(Action typeAction,
     String startDate, String endDate, List<String> ownerIds, int limit) {
 
+    final String startDateParameter = "actions.jsonb->>'dateAction' >= $4";
+    final String endDateParameter = "actions.jsonb->>'dateAction' < $5";
+    final String actionsParameter ="WHERE actions.jsonb->>'typeAction' IN ($2, $3)";
+
     String ownerIdsFilter = buildOwnerIdsFilter(ownerIds);
+    Tuple params = null;
+
+    List<String> dateParams = new ArrayList<>();
+    dateParams.add(actionsParameter);
```

Comment:
Here it's obvious that dateParams is not really about date. Let's rename it to `whereConditions`.